app/models/note.rb

```diff
@@ -34,6 +34,16 @@ class Note < ApplicationRecord
 
   scope :visible, -> { where.not(:status => "hidden") }
   scope :invisible, -> { where(:status => "hidden") }
+  scope :with_status, lambda { |status|
+    case status
+    when "open"
+      where(:status => "open")
+    when "closed"
+      where(:status => "closed")
+    else
+      all
+    end
+  }
```

Comment:
Do we even need the scope? A similar scope for issues is not used by the issues controller:

https://github.com/openstreetmap/openstreetmap-website/blob/e15a92a30213d87e10e000a240a7dcab66c208b4/app/controllers/issues_controller.rb#L37

Reply:
Fixed and updated the PR. Since this is the only place using the scope, I removed it to simplify the code and maintain consistency. If a need arises, we can reintroduce it later.

Also, I’ve added a test in the controller to cover the basic status filtering functionality. Let me know if further tests are needed.